src/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -33,12 +33,27 @@
  * @author tfennell
  */
 @CommandLineProgramProperties(
-        usage = "Computes a number of metrics that are useful for evaluating coverage and performance of " +
-                "whole genome sequencing experiments.",
-        usageShort = "Writes whole genome sequencing-related metrics for a SAM or BAM file",
+        usage = CollectWgsMetrics.USAGE_SUMMARY + CollectWgsMetrics.USAGE_DETAILS,
+        usageShort = CollectWgsMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CollectWgsMetrics extends CommandLineProgram {
+    static final String USAGE_SUMMARY = "Computes a number of metrics that are useful for evaluating coverage and" +
+            " performance of whole genome sequencing experiments";
+    static final String USAGE_DETAILS = "CollectWgsMetrics is a tool that computes a number of metrics that are useful " +
+            "for evaluating coverage and performance of whole genome sequencing (WGS) experiments.  Metrics" +
```

Comment:
Although I realize now that that is the case for a lot of these; it's not a big deal. Maybe shorten the summary slightly but don't spend much effort on this.